Description: Using absolute_import
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-06-19

--- python-oslo.messaging-1.14.0.orig/oslo_messaging/tests/__init__.py
+++ python-oslo.messaging-1.14.0/oslo_messaging/tests/__init__.py
@@ -14,6 +14,7 @@
 #    under the License.
 
 # Import oslotest before importing test submodules to setup six.moves for mock
+from __future__ import absolute_import
 import oslotest
 
 try:
